lib/filters.js

```diff
@@ -1,14 +1,96 @@
 'use strict';
 
 var transformers = require('transformers');
+var jstransformer = require('jstransformer');
+var uglify = require('uglify-js');
+var CleanCSS = require('clean-css');
+
+var warned = {};
+var alternatives = {
+  uglifyJS: 'uglify-js',
+  uglify: 'uglify-js',
+  uglifyCSS: 'clean-css',
+  'uglify-css': 'clean-css' ,
+  uglifyJSON: 'json',
+  'uglify-json': 'json',
+  live: 'livescript',
+  LiveScript: 'livescript',
+  ls: 'livescript',
+  // TODO: remove if we add support for coffeekup
+  coffeekup: 'coffeecup',
+  // The `style` transformer is not the same as the `stylus` jstransformer
+  styl: 'stylus',
+  coffee: 'coffee-script',
+  coffeescript: 'coffee-script',
+  coffeeScript: 'coffee-script',
+  // these marker transformers haven't made sense in a long time
+  css: 'verbatim',
+  js: 'verbatim',
+};
```

Comment:
We could, but I think in this case it's probably better to get people to specify the exact canonical name of that transform.  It prevents any potential ambiguity and makes loading the relevant transformer much simpler (otherwise we have to look for all possible transformers in the parent path and load all of them just in case one of them matches).